cmd/images-back/main.go

```diff
@@ -10,29 +10,66 @@ import (
 	"os/signal"
 	"strconv"
 	"time"
+	_ "github.com/lib/pq"
+	"database/sql"
 )
 
 var port int
+var db string
 
 func init() {
-	defPort := 5000
+	defPort := 8080
+	var defDB string
 	if portVar, ok := os.LookupEnv("PORT"); ok {
 		if portValue, err := strconv.Atoi(portVar); err == nil {
 			defPort = portValue
 		}
 	}
+	if dbVar, ok := os.LookupEnv("DATABASE_URL"); ok {
+		defDB = dbVar
+	}
 	flag.IntVar(&port, "port", defPort, "port to listen on")
+	flag.StringVar(&db, "db", defDB, "database to connect to")
 }
 
 func main() {
 	flag.Parse()
 	e := echo.New()
-	e.Static("/static", "static")
 	e.Logger.SetLevel(log.INFO)
+
 	e.GET("/", func(c echo.Context) error {
 		return c.String(http.StatusOK, "Hello, World!")
 	})
-	e.File("/index", "static/index.html")
+
+	e.GET("dbTest", func(c echo.Context) error {
+		//connStr := "user=images password=images_go dbname=imagesapp sslmode=disable"
+		url := db
+		dbPsql, err := sql.Open("postgres", url)
+		if err != nil {
+			log.Fatal(err)
+		}
+		defer dbPsql.Close()
+
+		err = dbPsql.Ping()
+		if err != nil {
+			panic(err)
+		}
+		log.Print("DB OK!")
+		_, err = dbPsql.Exec("CREATE TABLE IF NOT EXISTS images_table3 (" +
```

Comment:
It is awesome, that you added code for creating the table. But lets not add it in this pull request. Instead we will add migrations functionality in the next task, ok?